react/StoreWrapper.js

```diff
@@ -174,6 +175,7 @@ class StoreWrapper extends Component {
             <ExtensionPoint id="highlight-overlay" />
           </NoSSR>
         )}
+        <GoogleSearchBox />
```

Comment:
This is adding the block for everyone, waiting for @klzns guidelines because he has more context on what is happening

Reply:
That's it

Reply:
Thanks